.github/workflows/workflow-deploy-infra.yml

```
@@ -102,14 +102,13 @@ jobs:
           scope: subscription
-          template: ./.azure/infrastructure/main.bicep
-          subscriptionId: ${{ secrets.AZURE_SUBSCRIPTION_ID }}
-          deploymentMode: Incremental
-          deploymentName: dp-be-${{ inputs.environment }}-${{ inputs.version }}
-          region: ${{ inputs.region }}
-          failOnStdErr: false
-          additionalArguments: '--what-if'
-          parameters: ./.azure/infrastructure/${{ inputs.environment }}.bicepparam
+          type: deployment
+          operation: whatIf
+          name: dp-be-${{ inputs.environment }}-${{ inputs.version }} 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)